pkg_resources/tests/test_pkg_resources.py

```diff
@@ -219,8 +219,8 @@ def test_get_metadata__bad_utf8(tmpdir):
         "codec can't decode byte 0xe9 in position 1: "
         'invalid continuation byte in METADATA file at path: '
     )
-    assert expected in actual, 'actual: {}'.format(actual)
-    assert actual.endswith(metadata_path), 'actual: {}'.format(actual)
+    assert expected in actual, f'actual: {actual}'
+    assert actual.endswith(metadata_path), f'actual: {actual}'
```

Comment:
Technically changes the string, to simplify further, but it's an assert info message


```suggestion
    assert expected in actual, f'{actual=}'
    assert actual.endswith(metadata_path), f'{actual=}'
```